external/js/cothority/src/byzcoin/beautifier/config.ts

```diff
@@ -0,0 +1,59 @@
+import { Darc } from "../../darc";
+import { Roster } from "../../network";
+import { Argument } from "../index";
+import { ChainConfig } from "../index";
+import { IArgsSchema } from "./utils";
+// tslint:disable-next-line
+const varint = require("varint");
```

Comment:
I never know - will any of the following work?
```
import * as varint from "varint";
import varint from "varint";
```
???

Reply:
> I never know - will any of the following work?
unfortunately no : /